src/routes/get-started/ReportBug.tsx

```diff
@@ -180,6 +206,26 @@ export default function ReportBugPage() {
     return valid;
   }
 
+  useEffect(() => {
```

Comment:
You could add the `async` to line `209`. The only downfall is if you do you can't return the "cleanup" function, but in this case you don't have one, so it's fine. Then you don't have to create a function and manually call it.

Reply:
I don't think this is correct, `useEffect` is either supposed to return nothing or a clean-up function as you said, it's not supposed to return a Promise, which is what an async function returns. I could turn the inside into a self-invoking function, but that'll only remove one line of code, and I think makes things slightly less readable, thoughts?

Reply:
When you call the async function as I mentioned it is returning nothing (it's not returning a promise), so it is ok. As I mentioned if you need to return a cleanup then it doesn't work and you would have to do something like you are doing.